code/game/jobs/job/research.dm

```diff
@@ -149,15 +149,65 @@
 	)
 
 	skill_points = 20
-	roleplay_difficulty = "Easy - Medium"
+	roleplay_difficulty = "Medium - Hard"
 	mechanical_difficulty = "Medium"
 
 /datum/job/seniormentalist/equip(mob/living/carbon/human/H)
-	if (H.mind?.role_alt_title == "Senior Mentalist") //Basic level of Psionics, with ability to read others.
-		psi_faculties = list("[PSI_REDACTION]" = PSI_RANK_OPERANT)
-		psi_faculties = list("[PSI_COERCION]" = PSI_RANK_MASTER)
-		psi_faculties = list("[PSI_PSYCHOKINESIS]" = PSI_RANK_OPERANT)
-		psi_faculties = list("[PSI_ENERGISTICS]" = PSI_RANK_OPERANT)
+	if (H.mind?.role_alt_title == "Senior Mentalist") //Higher level of Psionics, as Seniormost expert on them in the site.
+		psi_faculties = list(
+			"[PSI_COERCION]" = PSI_RANK_MASTER,
+			"[PSI_PSYCHOKINESIS]" = PSI_RANK_OPERANT,
+			"[PSI_REDACTION]" = PSI_RANK_OPERANT,
+			"[PSI_ENERGISTICS]" = PSI_RANK_OPERANT
+			)
+	return ..()
```

Comment:
```suggestion
	. = ..()
	psi_faculties = list(
		"[PSI_COERCION]" = PSI_RANK_MASTER,
		"[PSI_PSYCHOKINESIS]" = PSI_RANK_OPERANT,
		"[PSI_REDACTION]" = PSI_RANK_OPERANT,
		"[PSI_ENERGISTICS]" = PSI_RANK_OPERANT
	)
```
sorry but locking it behind the alt title is genuinely dumb at this point
also style fixes